app/javascript/components/UI/QuestionButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {NavLink} from "react-router-dom";
+
+const QuestionButton = () => {
+  return (
+    <NavLink className="icon-circle" style={{position: 'fixed', right: 47, top: 756 }} to="mailto: support@vibereport.app">
+      <svg className="svg-inline--fa fa-question fa-3x" aria-hidden="true" focusable="false" data-prefix="fas"
```

Comment:
Create/add  pls new SVG file to assets/images folder and use it here.

Reply:
done